Form/Type/PhoneNumberType.php

```diff
@@ -64,10 +64,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 if (false === isset($countries[$region])) {
                     continue;
                 }
+                
+                if (null !== $options['country_widget_label'] && is_callable($options['country_widget_label'])) {
+                    $countryChoices[$options['country_widget_label']($countries, $name, $region)] = $region;
+                } else {
+                    $countryChoices[sprintf('%s (+%s)', $name, $countries[$region])] = $region;
+                }
```

Comment:
@xabbuh Is a default `sprintf()` call okay, in order to get rid off the else branch?